corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -26,6 +28,11 @@ def get_valid_recipients(recipients):
     """
     from corehq.util.models import BouncedEmail
     bounced_emails = BouncedEmail.get_hard_bounced_emails(recipients)
+    for bounced_email in bounced_emails:
+        email_domain = bounced_email.split('@')[1]
```

Comment:
Could bounced_email ever not have a `@` in it? seems worth catching that possibility unless we're quite sure

Reply:
updated